CondCore/Utilities/src/CondDBFetch.cc

```diff
@@ -140,9 +140,10 @@ namespace cond {
       FETCH_PAYLOAD_CASE( HBHENegativeEFilter )
       FETCH_PAYLOAD_CASE( HcalChannelQuality )
       FETCH_PAYLOAD_CASE( HcalCholeskyMatrices )
+      FETCH_PAYLOAD_CASE( HcalDcsValues )
       FETCH_PAYLOAD_CASE( HcalElectronicsMap )
       FETCH_PAYLOAD_CASE( HcalFlagHFDigiTimeParams )
-      FETCH_PAYLOAD_CASE( HcalDcsValues )
+      FETCH_PAYLOAD_CASE( HcalFrontEndMap )
       FETCH_PAYLOAD_CASE( HcalGains )
```

Comment:
@bsunanda thanks, this will allow to use the condb_import macro on the new CondFormat